lib/disk-utils.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
```

Comment:
A few low level promisified file-system functions, these bits seemed to get in the way (for me at least) when reading the plug-in logic itself so I moved them here.